packages/govtnz-ds/src/normalize-govuk.ts

```diff
@@ -876,8 +924,20 @@ const govUKToGovtNZCSS = async (oldCSS: string) => {
     [
       'all',
       '.g-button',
-      'border-radius: 2px; font-weight: 500; line-height: 1.2; color: black; padding: 16px; padding-top: 16px; padding-bottom: 16px; box-shadow:none'
+      `
+        padding: 16px;
+        padding-top: 16px;
+        padding-bottom: 16px;
+        font-family: National2;
```

Comment:
I think our DS font should be `Arial, sans-serif` (ie. system fonts), however the website then overrides that with National2.